src/components/views/rooms/SendMessageComposer.js

```diff
@@ -88,6 +90,25 @@ export function createMessageContent(model, permalinkCreator, replyToEvent) {
     return content;
 }
 
+// exported for tests
+export function isQuickReaction(model) {
+    const parts = model.parts;
+    if (parts.length == 0) return false;
+    let text = parts[0].text;
+    text += parts[1] ? parts[1].text : "";
+    // shortcut takes the form "+:emoji:" or "+ :emoji:""
+    // can be in 1 or 2 parts
+    if (parts.length <= 2) {
+        const hasShortcut = text.startsWith("+") || text.startsWith("+ ");
+        const emojiMatch = text.match(EMOJI_REGEX);
```

Comment:
to get the text for the model, you can use `textSerialize(model)` in `editor/serialize` which should be more future proof in case we change how a given input is split up in parts.

Reply:
Nice, will do this!